inspec.yml

```diff
@@ -0,0 +1,195 @@
+name: cms-ars-5.0-oracle-mysql-8-stig-overlay
+title: Oracle MySQL 8.0 STIG Automated Compliance Validation Profile
+maintainer: MITRE SAF Team
+copyright: MITRE, 2022
+license: Apache-2.0
+summary: "This Security Technical Implementation Guide is published as a tool to improve the security of Department of Defense (DoD) information systems. The requirements are derived from the National Institute of Standards and Technology (NIST) 800-53 and related documents. Comments or proposed revisions to this document should be sent via email to the following address: disa.stig_spt@mail.mil."
+version: 1.1.0
+inspec_version: ">= 4.0"
+
+depends:
+  - name: oracle-mysql-8-stig-baseline
+    url: https://github.com/mitre/oracle-mysql-8-stig-baseline/archive/main.tar.gz
+
+inputs:
```

Comment:
Looks like `system_categorization` input is missing. Also, the License file is empty

Reply:
Updated License file (hope it's right!))

Reply:
It looks good until the bullets. @ejaronne do we still need the last 3 paragraphs from [this License.md](https://github.com/CMSgov/cms-ars-5.0-microsoft-windows-server-2019-stig-overlay/blob/main/LICENSE.md)?